scripts/nqp/actions/skills.py

```diff
@@ -223,3 +248,74 @@ def build_effects(self, entity: EntityID) -> List[DamageEffect]:
     def get_animation(self, aesthetic: Aesthetic):
         # we can show animations depending on the direction with self.direction
         return aesthetic.sprites.attack
+
+
+@data_defined_skill
```

Comment:
The decorator `@data_defined_skill` calls the function set_properties on the class so we don't have to repeat the code that loads the properties from the library on every skill.

Reply:
By the way, doing this

```
@function_decorator
class A:
     def __init__():
         pass
```
is equivalent to writing this
```
class A:
     def __init__():
         pass
A = function_decorator(A)
```